doc/py_tutorials/py_setup/py_setup_in_ubuntu/py_setup_in_ubuntu.markdown

```diff
@@ -86,12 +106,11 @@ But it may be a little old.
 If you want to get latest libraries, you can install development files for system libraries of these formats.
 
 ```
-sudo apt-get install libpng-devel
-sudo apt-get install libjpeg-turbo-devel
-sudo apt-get install jasper-devel
```

Comment:
BTW, Jasper is available in Ubuntu 16.04, but it has been dropped from Ubuntu 18.04.